### ISSUE TITLE
Make "Terminate Running Task" Automatically Start Requested Task

### ISSUE THREAD

**Issue by BobChao87** [Microsoft/vscode#19646]
**Feature request:**

When warned that  "There is an active running task right now. Terminate it first before executing another task." when trying to start one task while another is running, selecting "Terminate Running Task" would terminate the running task and start the requested task.

**What it does currently:** 
Terminates the task and then the user must redeclare which task they want to run, which (aside from the build task) means pulling up the task runner menu and selecting the task again.

**Exceptions/Special Cases:**
If there exists a way to bring up this warning dialog besides attempting to start another task, the behavior should remain the same as it is now and not attempt to start any task as there is no task requested by the user.

**Why we should add this:**
This will reduce effort required to terminate a running build task and move directly into another. This most often occurs when a programmer realizes they started a task with a bug and need to restart it or when terminating a watch task for some other task, which seems like it would be a pretty common use case.

This change will also, IMHO, fulfill the what the user would initially expect the "Terminate Running Task" button to do. This will reduce menu navigation and generally speed up development for anyone who would see this dialog.

**Comment by dbaeumer** [comment]
The new task runner execution engine which will use the terminal will allow executing more than one task in parallel. So there will be no need to terminate the previous one.

However the question will then be if we need to support limiting tasks to one.

**Comment by dbaeumer** [comment]
@NCC1701M it is available since Feb, but you need to opt in using the `"version": "2.0.0"` in the tasks.json.

**Comment by dbaeumer** [comment]
The new runner allows to execute n tasks in parallel. There is also a feature request to allow to run the same task n times. Closing this issue.
